Topics/Professionalism.md

```diff
@@ -1,75 +1,155 @@
+   
+6. Invest in reliable technology: Ensure that your computer, internet connection, and other essential tools are up-to-date and reliable to avoid technical issues during work.
 
 ### Client interactions
-1 Active listening: Pay attention to the client's needs and concerns, and ask clarifying questions to ensure a clear understanding of their requirements.
+
+1. Active listening: Pay attention to the client's needs and concerns, and ask clarifying questions to ensure a clear understanding of their requirements.
+   
 2. Professional appearance: Dress appropriately for video calls and ensure your background is tidy and uncluttered to present a professional image.
+   
 3. Setting realistic expectations: Communicate the scope, timeline, and potential challenges of a project upfront to avoid miscommunications and manage expectations.
+   
 4. Provide regular updates: Keep clients informed about project progress, milestones, and any changes or obstacles that may affect the project's outcome.
+   
 5. Delivering high-quality work: Strive for excellence in your work, thoroughly test your code, and address any issues promptly to ensure client satisfaction.
+    
 6. Handle difficult situations with grace: If conflicts or misunderstandings arise, remain calm, listen to the client's concerns, and work collaboratively to find a solution.
 
 ### Additional practices:
-You can find additional practices on this website: https://engineerscanada.ca/professional-practice-in-software-engineering
 
-By implementing these best practices, you can maintain a professional demeanor in various situations, enhancing your reputation as a software engineer and fostering positive relationships with colleagues and clients.
+You can find additional professional practices on this website: https://engineerscanada.ca/professional-practice-in-software-engineering
+
+By implementing these best practices, you can maintain a professional demeanour in various situations, enhancing your reputation as a software engineer and fostering positive relationships with colleagues and clients.
+
+# Business Communication as it Applies to Professionalism
+
+Another very important professional skill for software engineers is the ability to communicate technical information to a non-technical audience. Often, projects in software engineering will involve stakeholders or managers/team members with varying technical knowledge regarding the product. These folks want to understand the impact that your technical contributions have had on the project, what the new features are and how they work from a user's perspective. Software is not nearly as useful if it cannot be sold to the people who need it, so it is essential to know how to perform business communication with non-technical stakeholders or co-workers.
+
+Here are seven tips from a Lucidchart article (5) on communicating technical information to a non-technical audience:
```

Comment:
I agree that communicating technical information to non-technical audiences is very important and I am glad you covered this topic!

Reply:
This is something that I did not learn the importance of until I began speaking to people in industry. I am hoping this will be practical and new information for many CSC301 students!